variables.tf

```diff
@@ -70,11 +70,24 @@ variable "apply_immediately" {
   description = "Specifies whether any database modifications are applied immediately, or during the next maintenance window"
 }
 
+variable "availability_zone" {
+  type        = string
+  default     = ""
+  description = "List of Availability Zones for the cluster"
```

Comment:
Looks like this description is copy / pasta. Mind updating? 

Also, what happens if a module consumer doesn't pass a value for `availability_zone`? Is that acceptable behavior by the provider?

Reply:
yes, `availability_zone` can just be an empty string. Same with `availability_zones`, an empty list is accpetable.

Reply:
Updated.